packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -719,6 +726,7 @@ export class MgtPerson extends MgtTemplatedComponent {
     // this reduces the redundant announcement of the user's name.
     const textTemplate = html`
       <span 
+        title="${ifDefined(imageOnly ? titleText : undefined)}"
```

Comment:
I don't think that this is the right check.

There are a bunch of configs that can cause initials to show:
- https://mgt.dev/?path=/story/components-mgt-person-properties--person-disable-image-fetch
- https://mgt.dev/?path=/story/components-mgt-person-properties--person-avatar-type
- Or just not having a photo uploaded....

I think that we should modify the logic that sets imageOnly to only check for the view being ViewType.image the distinction between avatar/initials doesn't need to be taken into account for this purpose AFAICT

Reply:
That's true. Updating the logic for the initials to only show a title if the are image representations.